README.md

```diff
@@ -1 +1,21 @@
-# elasticsearch-admin-scripts
\ No newline at end of file
+# elasticsearch-admin-scripts
+
+This repo stores the Cerebro team's scripts that make it easier to manage Elasticsearch. Detailed descriptions are in the scripts themselves.
+
+## Example usage
+```bash
+./createIndex.sh --user "username" --env "local" --index "myIndex" --docMappingFile "document_mapping.json"
+```
+
+Every script requires two arguments:
+* `env`: the environment of the cluster the script is run against (e.g. `oscf-dev`)
+* `user`: a user name with access to the cluster in `env`
```

Comment:
Is this like "brichardson"? Do we need password?

Reply:
`brichardson` isn't a user that has access to any of the non-monitoring clusters. I didn't want to document credentials in this project. I can add a note that says you'll be prompted for the password when you run any of these.